aspnetcore/migration/22-to-30.md

```diff
@@ -174,6 +174,8 @@ General advice:
 * Replace `UseMvc` or `UseSignalR` with `UseEndpoints`.
 * If the app uses [CORS](xref:security/cors) scenarios, such as `[EnableCors]`, place the call to `UseCors` before any other middlewares that use CORS (for example, place `UseCors` before `UseAuthentication`, `UseAuthorization`, and `UseMvc`).
 * Replace `IHostingEnvironment` with `IWebHostEnvironment` and add a `using` statement for the `Microsoft.Extensions.Hosting` namespace.
+* Replace `IApplicationLifetime` with `IHostApplicationLifetime`.
+* Replace `EnvironmentName` with `Environments`.
```

Comment:
Include the namespaces for clarity. Many of these replacements are in different namespaces.